MaterialComponentsBeta.podspec

```diff
@@ -104,6 +104,23 @@ Pod::Spec.new do |mdc|
     end
   end
 
+  mdc.subspec "Tabs+TabBarView" do |extension|
+    extension.ios.deployment_target = '9.0'
+    extension.public_header_files = "components/#{extension.base_name.split('+')[0]}/src/#{extension.base_name.split('+')[1]}/*.h"
+    extension.source_files = [
+      "components/#{extension.base_name.split('+')[0]}/src/#{extension.base_name.split('+')[1]}/*.{h,m}",
+      "components/#{extension.base_name.split('+')[0]}/src/#{extension.base_name.split('+')[1]}/private/*.{h,m}"
+    ]
+    extension.dependency "MaterialComponents/#{extension.base_name.split('+')[0]}"
+
+    extension.test_spec 'UnitTests' do |unit_tests|
+      unit_tests.source_files = [
+      "components/#{extension.base_name.split('+')[0]}/tests/unit/#{extension.base_name.split('+')[1]}/*.{h,m,swift}",
+      "components/#{extension.base_name.split('+')[0]}/tests/unit/#{extension.base_name.split('+')[1]}/supplemental/*.{h,m,swift}"
```

Comment:
Indent these two lines please.

Reply:
That doesn't seem correct - most of the rest of the file has the same indentation.

Reply:
This discussion is now obsolete for this PR (since I copied a different section of the Podspec for the new component), but I've opened [an issue to track the suboptimal indentation in the Beta podspec file](https://github.com/material-components/material-components-ios/issues/7638).